apps/docs/src/app/examples/checkbox/customicons/customicons.component.html

```diff
@@ -0,0 +1,11 @@
+<anglify-checkbox [checked]="true" [rippleOrigin]="'center'">Check me!</anglify-checkbox>
+<anglify-checkbox [checked]="true" [ripple]="false">No ripple!</anglify-checkbox>
+<anglify-checkbox [checked]="true">
```

Comment:
```suggestion
<anglify-checkbox checked>
```

This works because of the BooleanLike type. Please apply this everywhere